wca-regulations.md

```diff
@@ -200,6 +200,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
         - 7f1d) Surface: The flat surface on which the Stackmat has been placed. The mat is considered a part of the surface. The timer is not considered a part of the surface.
     - 7f2) The Stackmat timer must be attached to the mat and placed on the surface, with the timer on the side of the mat nearest to the competitor.
         - 7f2a) Exception: For Solving With Feet, the Stackmat must be placed directly on the floor. The timer device may be placed on the side of the mat farthest from the competitor.
+    - 7f3) The mat must have a line drawn 3 cm away from the center point of the timer, measuring 30 cm in length, and parallel to the tangent on the center point.
```

Comment:
I would say "marked" instead of "drawn" so that this can be temporarily done using tape.

"and parallel to the tangent on the center point." doesn't really make sense to me, though. How about "horizontal and symmetric from left to right from the competitor's perspective"?